sentry-android-core/src/test/java/io/sentry/android/core/internal/util/PrivilegeEscalationViaContentProviderCheckerTest.kt

```diff
@@ -0,0 +1,105 @@
+package io.sentry.android.core.internal.util
+
+import android.content.ContentProvider
+import android.content.Context
+import android.os.Build
+import io.sentry.android.core.BuildInfoProvider
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.verifyNoInteractions
+import org.mockito.kotlin.whenever
+import kotlin.test.Test
+import kotlin.test.assertNotNull
+import kotlin.test.assertNull
+
+class PrivilegeEscalationViaContentProviderCheckerTest {
```

Comment:
I used one of the existing tests to make sure I'm following the code style/convention.